### ISSUE TITLE
Web Standards and Accessibility Development Guide missing?

### ISSUE THREAD

**Issue by datasmithtechgrp** [bcgov/design-system#439]
### Related component

all

### Description

Where is the Web Standards and Accessibility Development Guide that our web apps are to follow ? 

-  

### Rationale

The Guide, which is intended for both expert users and those with minimal web experience, includes both quick reference points and more detailed explanations of web development topics (such as font usage, text size, etc.), as well as examples of how to properly apply them to web pages.

### Additional information

https://www.queensu.ca/accessibility/across-campus/web-standards-and-accessibility-development-guide
https://webaccess.msu.edu/tutorials

**Comment by mkernohanbc** [comment]
HI @datasmithtechgrp, thanks for reaching out. 

There are a few different resources available: 

1. For software development: [B.C. Developer Guide (developer.gov.bc.ca)](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide?utm_source=common-resources-link&utm_medium=web&utm_campaign=digital)
2. For accessibility guidance: [Simplified WCAG guidance (digital.gov.bc.ca)](https://digital.gov.bc.ca/wcag/home/) — accessibility
3. For content design and writing: [Web Style Guide (gov.bc.ca)](https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide)
4. For visual branding: [B.C. Visual Identity Program (gov.bc.ca)](https://www2.gov.bc.ca/gov/content/governments/services-for-government/policies-procedures/bc-visual-identity)

Within the [Foundations section in the B.C. Design System docs](https://www2.gov.bc.ca/gov/content/digital/design-system/foundations), you'll also find guidance for basic concepts like typography, colour and layout. 

Hope that helps!

**Comment by datasmithtechgrp** [comment]
@mkernohanbc  Can you please refer me to an angular project repo that uses approved styles?
